dataland-frontend/src/utils/indexMapper.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Map the backend indices to frontend indices
+ * adopted from https://riptutorial.com/javascript/example/8628/merge-two-array-as-key-value-pair
+ *
+ */
+import apiSpecs from "@/../build/clients/backend/backendOpenApi.json";
+
+export function stockIndexObject() {
+    const frontend = [
+        "CDAX",
+        "DAX",
+        "General Standards",
+        "GEX",
+        "MDAX",
+        "Prime Standards",
+        "SDAX",
+        "TecDAX",
+        "ScaleHDAX",
+        "DAX 50 ESG"
+    ];
+    const backend = apiSpecs.components.schemas.CompanyInformation.properties["indices"].items.enum
+    return frontend.reduce(function (result: any, field, index) {
```

Comment:
Hier finde ich das fehlerhaft wenn die reinfolge oder was anderes in backend sich ändert. Ich würde den StringHumanizer hier verwenden

Reply:
Und ich finde auch hier, dass `backend` ein zu allgemeiner Name für die Variable ist.

Als Leser musste ich im Frontend-Code (das passiert auch an anderen Stellen häufig) immer wieder zur Variablen-Deklaration hochschauen um zu verstehen, was hinter `backend` steckt.

Reply:
die gesamte datei ist weg, da nicht mehr gebraucht